### PR TITLE
Update news for v4.9

### PR DESCRIPTION
**Pull request by calvinlu3**
This is related to https://github.com/oncokb/oncokb/issues/3570

- [x] Update news
- [x] Update oncokb_summary.png

### CODE REVIEW

**Review by zhx828 — CHANGES_REQUESTED** [review]
Could you change the date to 10/2?

File: src/main/webapp/app/pages/newsPage/NewsPageContent.tsx [hunk 1]
```diff
@@ -132,6 +135,16 @@ export const CHANGED_ANNOTATION_ADDITIONAL_DRUG_SAME_LEVEL_COLUMNS = [
   { name: EVIDENCE },
 ];
 
+export const CHANGED_ANNOTATION_ADDITIONAL_DRUG_SAME_LEVEL_COLUMNS_NEW = [
```
Comment (zhx828) [inline src/main/webapp/app/pages/newsPage/NewsPageContent.tsx:138]: What's the difference of using CHANGED_ANNOTATION_ADDITIONAL_DRUG_SAME_LEVEL_COLUMNS?

Comment (calvinlu3) [reply src/main/webapp/app/pages/newsPage/NewsPageContent.tsx:138]: The only difference is the column name that we use:
New `Drug(s) currently in OncoKB   |  Drug(s) added to OncoKB`
vs.
Old `Drug(s) Already in OncoKB   |   Newly Added Drug(s)`

Comment (calvinlu3) [reply src/main/webapp/app/pages/newsPage/NewsPageContent.tsx:138]: The team wants to use the new language for the header names and thinks its fine to keep the old language for the older releases.

File: src/main/webapp/app/pages/newsPage/ChangedAnnotatonListItem.tsx [hunk 2]
```diff
@@ -48,7 +48,7 @@ export const ChangedAnnotationListItem = (props: {
       annotationColumnHeader = CHANGED_ANNOTATION_ADDITIONAL_DRUG_SAME_LEVEL_COLUMNS;
       useOneLineRowClass = false;
       defaultTitle =
-        'Updated therapeutic implications - addition of therapies for variants with a level of evidence';
+        'Updated therapeutic implications - Addition of therapies for variants with a level of evidence';
```
Comment (calvinlu3) [inline src/main/webapp/app/pages/newsPage/ChangedAnnotatonListItem.tsx:51]: New change

**Review by zhx828 — APPROVED** [review]
LGTM thanks!

**Pull request merged**